src/main/kotlin/pt/up/fe/ni/website/backend/controller/ErrorController.kt

```diff
@@ -40,7 +42,9 @@ class ErrorController : ErrorController {
                 SimpleError(
                     violation.message,
                     violation.propertyPath.toString(),
-                    violation.invalidValue
+                    violation.invalidValue?.let {
+                        if (isSerializable(it)) it else null
+                    }
```

Comment:
Really minor, but I think using `takeIf(::isSerializable)` instead of `let` would be more idiomatic in this case.